README.md

```diff
@@ -2,6 +2,8 @@
 
 **Latest Version: 1.7.2**
 
+_**Notice**: The Carbon Black Cloud portion of CBAPI has moved to https://github.com/carbonblack/carbon-black-cloud-sdk-python. Any future development and bug fixes for Carbon Black Cloud APIs will be made there. Carbon Black EDR and App Control will remain supported at CBAPI_
+
 These are the new Python bindings for the Carbon Black Enterprise Response and Enterprise Protection REST APIs.
```

Comment:
They're probably not "new" bindings at this stage.  Change the sentence to
"These are the Python bindings..."